deploy/platform/redhat/redhat_docker_build.sh

```diff
@@ -23,6 +23,10 @@ do_createrepo() {
   fi
   : && createrepo -q $update_args -o ${tmpdir} ${repodir}/${FLAVOR}/RPMS
   checkstatus abort "Failure: Problem creating rpm repository in ${repodir}!" $?
+  rm -f ${tmpdir}/repodata/repomd.xml.asc
+  export GNUPGHOME=/sign_keys/.gnupg
```

Comment:
Instead of `export` and then `unset`, you can just set that env for just the `gpg` command, e.g.
`GNUPGHOME=/sign_keys/.gnupg gpg --local-user MDSplus...`

Reply:
A-OK, will make that change right now.